.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: python
 python:
   - 3.5
+services:
+  - postgresql
 env:
   - TRAVIS_NODE_VERSION="4.4"
 install:
   - pip install -r requirements.txt
-  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION
-  - npm install
 script:
-  - echo 'no scripts yet!'
+  - python manage.py test --nomigrations
```

Comment:
For now I am passing `--nomigrations` so that WIP PRs that haven't yet added a migration will work. However, ideally `--nomigrations` would depend on the branch being tested, since we *do* always want `master` to have up-to-date migrations.